src/vue/components/inflections-table-wide.vue

```diff
@@ -3,29 +3,29 @@
   <div>
     <div @click="collapse"
         class="alpheios-inflections__title alpheios-clickable">
-      {{view.title}}
+      {{title}}
       <span v-show="state.collapsed">[+]</span>
       <span v-show="!state.collapsed">[-]</span>
     </div>
 
     <template v-if="!state.collapsed">
       <h4
-          v-show="view.additionalTitle"
+          v-show="additionalTitle"
```

Comment:
You have `v-if` condition in the creating inflection tables, so each time when it collapsed again - all HTML DOM is destroyed, when you open again - it rerenders again. May be it would be good to eliminate  HTML DOM re-rendering here - as it could be rather a heavy operation - rendering table with text? What do you think?

Reply:
That's probably related to the other topic of speed vs memory usage. I think we should decide on the common approach between the two.

Reply:
We were talking about this approach here
https://github.com/alpheios-project/components/issues/366

And if I understood right, we decided to eliminate re-renders HTML DOM objects and use `v-show` for that cases. Because `v-show` changes only styles, but doesn't make browse to recreate DOM elements, when a user wants to see an inflection table, that was already rendered to the DOM.

What do you think @kirlat ?

Reply:
I agree that once the HTML of the table is generated, it should not be re-renderd each time the table is expanded. So there should be no `state.collapsed` in `v-if`s, it should be used in `v-show` only. I rearranged the code in a way that, I think, will solve this issue:
https://github.com/alpheios-project/components/blob/comp-i586-slow-activation/src/vue/components/inflections-table-wide.vue#L18-L84

@irina060981, is that what you was talking about?

Reply:
oh, yes - I am sorry - I wasn't able to add my comments to the line `v-if="!state.collapsed"`, github placed then my comment to the left panel.
That's why I tried to explain my thoughts in the comment in details.
Yes - now it is fixed!